apps/rush-lib/src/api/Rush.ts

```diff
@@ -63,12 +97,79 @@ export class Rush {
     return PackageJsonLookup.loadOwnPackageJson(__dirname).version;
   }
 
+  private static _tryGetRushConfiguration(): RushConfiguration | undefined {
+    try {
+      if (RushConfiguration.tryFindRushJsonLocation()) {
+        return RushConfiguration.loadFromDefaultLocation({ showVerbose: true });
+      }
+    } catch (e) {
+      // ignore
+    }
+
+    return undefined;
+  }
+
+  /**
+   * This function normalizes legacy options to the current {@see ILaunchOptions} object.
+   */
+  private static _normalizeLaunchOptions(arg: ILaunchOptions): ILaunchOptions {
+    return (typeof arg === 'boolean')
+      ? { isManaged: arg } // In older versions of Rush, this the `launch` functions took a boolean arg for "isManaged"
+      : arg;
+  }
+
   private static _printStartupBanner(isManaged: boolean): void {
+    interface IExtendedNodeProcess extends NodeJS.Process {
+      release: {
+        lts?: string;
+      };
+    }
+
+    const nodeVersion: string = process.versions.node;
+    const nodeMajorVersion: number = semver.major(nodeVersion);
+    const nodeReleaseLabel: string = (nodeMajorVersion % 2 === 0)
+      ? (!!(process as IExtendedNodeProcess).release.lts ? 'LTS' : 'pre-LTS')
+      : 'unstable';
+
     console.log(
       EOL +
       colors.bold(`Rush Multi-Project Build Tool ${Rush.version}` + colors.yellow(isManaged ? '' : ' (unmanaged)')) +
       colors.cyan(` - ${RushConstants.rushWebSiteUrl}`) +
+      EOL +
+      `NodeJS version is ${nodeVersion} (${nodeReleaseLabel})` +
       EOL
     );
   }
+
+  private static _warnAboutNodeVersion(rushConfiguration: RushConfiguration | undefined): void {
+    const nodeVersion: string = process.versions.node;
+
+    interface IExtendedNodeProcess extends NodeJS.Process {
+      release: {
+        lts?: string;
+      };
+    }
+
+    if (semver.satisfies(nodeVersion, '>= 11.0.0')) {
```

Comment:
Should `suppressNodeLtsWarning` apply to this warning as well?  My preference would be for it to suppress both warnings.

Reply:
Let's discuss this in https://github.com/microsoft/web-build-tools/issues/1373 where the design notes are.

Reply:
Discussed in the issue. Consensus is that we don't suppress the warning for any odd-numbered releases.

Reply:
Right, but I think Mike's point was that `>= 11.0.0` will match `12.0.0` as well, which is not intended.

This number should probably be increased to `>= 13.0.0`.

Reply:
Yeah I made that change as well. Take a look at the latest changes.